services/black/src/ambassador/BlackClient.cs

```diff
@@ -52,17 +33,16 @@ public BlackClient(string connectionString)
         public async Task SetGuestInformationAsync(SetGuestInformation guestInformation)
         {
             // ToDo I don't know if we should start and stop the endpoint in this call or do it outside
```

Comment:
i suggest creating the endpoint outside and keep it open for as long as the app is running

Reply:
I have to look that up in the particular docs. For now it should work.

Reply:
Haven't found anything useful, but I think we should depend the endpoint lifetime on the lifetime of the ambassador and make the ambassador disposable, so we can close the channel, unsubscribe from events etc.